Atomic.md

```diff
@@ -0,0 +1,618 @@
+    Fee: "40",
+    SigningPubKey: "03072BBE5F93D4906FC31A690A2C269F2B9A56D60DA9C2C6C0D88FB51B644C6F94",
+    TxnSignature: "30440220702ABC11419AD4940969CC32EB4D1BFDBFCA651F064F30D6E1646D74FBFC493902204E5B451B447B0F69904127F04FE71634BD825A8970B9467871DA89EEC4B021F8"
+  },
+  {
+    TransactionType: "Payment",
+    Account: "rUser1fcu9RJa5W1ncAuEgLJF2oJC6",
+    Destination: "rUser2fDds782Bd6eK15RDnGMtxf7m",
+    Amount: "6000000",
+    AtomicTxn: {
+      Account: "rUser1fcu9RJa5W1ncAuEgLJF2oJC6",
+      OuterSequence: 4,
+      Sequence: 10,
+      BatchIndex: 0
+    },
+    Sequence: 0,
+    Fee: "0",
+    SigningPubKey: "",
+    TxnSignature: ""
+  },
+  {
+    TransactionType: "Payment",
+    Account: "rUser2fDds782Bd6eK15RDnGMtxf7m",
+    Destination: "rUser1fcu9RJa5W1ncAuEgLJF2oJC6",
+    Amount: {
+      currency: "GKO",
+      issuer: "ruazs5h1qEsqpke88pcqnaseXdm6od2xc",
+      value: "2"
+    },
+    AtomicTxn: {
+      Account: "rUser1fcu9RJa5W1ncAuEgLJF2oJC6",
+      OuterSequence: 4,
+      Sequence: 20,
+      BatchIndex: 1
+    },
+    Sequence: 0,
+    Fee: "0",
+    SigningPubKey: "",
+    TxnSignature: ""
+  }
+]
+```
+</details>
+
+# Appendix
+
+## Appendix A: FAQ
+
+### A.1: What if I want a more complex tree of transactions that are AND/XORed together? Can I nest `Atomic` transactions?
+
+The original version of this spec supported nesting `Atomic` transactions. However, upon further analysis, that was deemed a bit too complicated for an initial version of `Atomic`, as most of the benefit from this new feature does not require nested transactions. Based on user and community need, this could be added as part of a V2.
+
+### A.2: What if all of the transactions fail? Will a fee still be claimed?
+
+Yes, just as they would if they were individually submitted. 
+
+### A.3: Could there be an additional atomicity type for allowing all transactions to be processed regardless of whether they succeeded?
+
+This is unnecessary, since that is equivalent to submitting the transactions normally.
+
+### A.4: Would this feature enable greater frontrunning abilities?
+
+That is definitely a concern. Ways to mitigate this are still being investigated. Some potential answers:
+* Charge for more extensive path usage
+* Have higher fees for atomic transactions
+* Submit the atomic transactions at the end of the ledger
+
+### A.5: What error is returned if all the transactions fail in an `OR`/`BATCH` transaction?
+
+A general error, `temBATCH_FAILED`/`tecBATCH_FAILED`, will be returned. A list of all the return codes encountered for the transactions that were processed will be included in the metadata, for easier debugging.
+
+### A.6: Can another account sign/pay for the outer transaction if they don't have any of the inner transactions?
+
+If there are multiple parties in the inner transactions, yes. Otherwise, no. This is because in a single party `Atomic` transaction, the inner transaction's signoff is provided by `AtomicSigners`.
```

Comment:
```suggestion
If there are multiple parties in the inner transactions, yes. Otherwise, no. This is because in a single party `Atomic` transaction, the inner transaction's signoff is provided by the normal transaction signing fields.
```